package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "serverless-kubeless",
-  "version": "0.8.3",
+  "version": "0.8.4",
```

Comment:
it's better to update the minor version since we are adding a new feature (0.9.0)

Reply:
done